### PR TITLE
(DOCSP-1089): Added known issue on GROUP BY.

### PR DESCRIPTION
**Pull request by atsansone**
@rychipman : This has been outstanding a while and I know that it has no resolution yet (BI-484), but could you take a look at this [added known issues page](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/anthonysansone/DOCSP-1089/reference/known-issues.html) and the first known issue listed to see if I have this correct?

### CODE REVIEW

File: source/reference/known-issues.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      +-------------+----------+---------------+-------------+-------------+
+      | sum(scores) | count(*) | count(scores) | max(scores) | min(scores) |
+      +-------------+----------+---------------+-------------+-------------+
+      |   NULL      |        0 |             0 |   NULL      |   NULL      |
+      +-------------+----------+---------------+-------------+-------------+
+ 
+   If you query the |bi-short| for this same data, it returns an 
+   empty data set. MongoDB returns nothing if no data exists for a 
+   value, its fields is not part of the document. Therefore, it cannot
```
Comment (nlarew) [inline source/reference/known-issues.txt]: This sentence doesn't really make sense.
`MongoDB returns nothing if no data exists for a value, its fields is not part of the document.`

File: source/reference/known-issues.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+(like ``SUM``, ``MIN``, or ``COUNT``) return ``NULL`` if there are no
+matching values for that aggregate function. This response is
+effectively an empty row.
+
+Given the exact same SQL query, the |bi-short| does not return
+``NULL`` for any aggregate functions; it returns an empty set instead.
+
+.. example::
+
+   Using Tableau, you query a MySQL database of students to find the
```
Comment (nlarew) [inline source/reference/known-issues.txt]: This example uses MySQL and MongoDB databases. We may want to remove this mention of MySQL and move it closer to the MySQL `NULL` result code block.

File: source/reference/known-issues.txt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+   sum, count, highest, and lowest test scores by each class.
+
+   Unknown to you, this is a new database table created for the new
+   classes of students, so there is no data in this table yet. This SQL
+   query would return no students because no scores had been entered:
```
Comment (nlarew) [inline source/reference/known-issues.txt]: I think we could split these two paragraphs in a different way:

```
You are analyzing the test performance of students for each of the
classes offered in the last year by a university. Using Tableau, you run
a SQL query to find statistics for each class, including the highest and
lowest test scores as well as the sum and count of all the scores in the
class.

.. code-block:: sql

   SELECT SUM(scores), COUNT(*), COUNT(scores), MAX(scores), MIN(scores)
   FROM students_2019
   GROUP BY class;

Unfortunately, you accidentally run the query on the data for next
year's classes, so there are no test scores for the query to find.

If the student data was stored in MySQL, this query would return a
single row with ``NULL`` values for fields that are calculated with the
``SUM``, ``MAX``, and ``MIN`` functions.

.. code-block:: sql

   SELECT SUM(scores), COUNT(*), COUNT(scores), MAX(scores), MIN(scores)
   FROM students_2019
   GROUP BY class;

   +-------------+----------+---------------+-------------+-------------+
   | sum(scores) | count(*) | count(scores) | max(scores) | min(scores) |
   +-------------+----------+---------------+-------------+-------------+
   |   NULL      |        0 |             0 |   NULL      |   NULL      |
   +-------------+----------+---------------+-------------+-------------+

If the student data was stored in MongoDB and accessed via the
|bi-short|, this query would return an empty set.

.. code-block:: sql
   
   SELECT SUM(scores), COUNT(*), COUNT(scores), MAX(scores), MIN(scores)
   FROM students_2019
   GROUP BY class;
   
   Empty set (0.00 sec)

Comment (atsansone) [reply source/reference/known-issues.txt]: Agreed. Fixed.

File: source/reference/known-issues.txt (hunk 1 above)
Comment (atsansone) [reply source/reference/known-issues.txt]: N/A. Fixed.

File: source/reference/known-issues.txt (hunk 2 above)
Comment (atsansone) [reply source/reference/known-issues.txt]: N/A. Fixed per suggestion below.

**Comment by atsansone** [comment]
Builds clean to `master`.

**Pull request merged**